api/README.md

```diff
@@ -1,30 +0,0 @@
-
-# OSF API
-
-## Getting started
-
-From the root osf directory:
-
-```bash
-pip install -r requirements.txt
-# Required for browse-able API
-python manage.py collectstatic
```

Comment:
Oh! I could have phrased my question better. I was referring specifically to the part about `python manage.py collectstatic`. Without it, the web-browsable API would look silly.